src/common/ImageAvatar.js

```diff
@@ -15,14 +15,17 @@ const styles = StyleSheet.create({
 });
 
 export default({ avatarUrl, size, status, isCircular, onPress }) => {
-  const imageStyle = {
+  const touchableStyle = {
     height: size,
     width: size,
-    borderRadius: isCircular ? size / 2 : size / 8,
+  };
+
+  const imageStyle = {
+    borderRadius: isCircular ? size / 2 : size / 8
   };
 
   return (
-    <Touchable onPress={onPress}>
+    <Touchable onPress={onPress} style={touchableStyle}>
       <Image
         style={imageStyle}
```

Comment:
I did it, but I guess simulator didn't reload it and I pushed without second reload - now I checked it again and you're right.